src/main/java/com/knu/buga1chuk/json/ReadPersonJsonDemo.java

```diff
@@ -0,0 +1,22 @@
+package com.knu.buga1chuk.json;
+
+import com.google.gson.Gson;
+import com.knu.buga1chuk.serialization.PersonList;
+import com.knu.buga1chuk.service.PersonJsonService;
+
+public class ReadPersonJsonDemo {
+    private static final String PEOPLE_DATA = "people.json";
+
+    public static void main(String[] args) {
+
+        PersonJsonService personJsonService = new PersonJsonService();
+
+        String result = personJsonService.getJsonPersonsFromFile(PEOPLE_DATA);
+        Gson gson = new Gson();
+
+        PersonList persons = gson.fromJson(result, PersonList.class);
+        System.out.println(persons);
```

Comment:
Implement the calculation of the youngest person logic. And remove useless souts.